electrumsv/tests/test_wallet.py

```diff
@@ -1,11 +1,14 @@
+import bitcoinx
```

Comment:
You're mixing third party dependency imports and standard library imports into some random extra section.